src/Squidex/app/shared/services/schemas.service.ts

```diff
@@ -333,6 +337,8 @@ export abstract class FieldPropertiesDto {
     public abstract formatValue(value: any): string;
 
     public abstract createValidators(isOptional: boolean): ValidatorFn[];
+
+    public abstract getDefaultValue(): any;
```

Comment:
A 5 minute request: Can you make this method not abstract, then we could get rid of a lot of dummy implementations.

Reply:
Good point. Fixed.